data/controls.js

```diff
@@ -4,34 +4,34 @@
  * http://mozilla.org/MPL/2.0/.
  */
 
-document.querySelector('.controls').style.display = 'none';
-document.querySelector('.video-wrapper').style.display = 'none';
+document.querySelector(".controls").style.display = "none";
```

Comment:
No reason at all. I think when I did the initial .eslintrc file, I tried both and we used double quotes 60% of the time and single quotes the other 40%.
I'm happy to switch everything to single quotes if you prefer. It should be a trivial fix with the ESLint `--fix` flag which will swap them all for us.